src/chocolatey.resources/helpers/functions/Get-CheckSumValid.ps1

```diff
@@ -189,27 +189,40 @@ Install-ChocolateyPackage
         throw "Checksum type '$checksumType' is unsupported. This type may be supported in a newer version of Chocolatey."
     }
 
-    $checksumExe = Join-Path "$helpersPath" '..\tools\checksum.exe'
-    if (!([System.IO.File]::Exists($checksumExe))) {
-        Update-SessionEnvironment
-        $checksumExe = Join-Path "$env:ChocolateyInstall" 'tools\checksum.exe'
-    }
-    Write-Debug "checksum.exe found at `'$checksumExe`'"
+    # Use Get-FileHash if available (added in Windows PowerShell 4.0)
+    if (Get-Command "Get-FileHash" -ErrorAction SilentlyContinue) {

```

Comment:
It probably would be significantly faster to check the PowerShell version, as `Get-Command` can be slow if the command does not exist (as it has to check the folders on the `$path` for native commands).